diff --git a/node_modules/circular-dependency-plugin/index.js b/node_modules/circular-dependency-plugin/index.js
index 32c7be9..245fa57 100644
--- a/node_modules/circular-dependency-plugin/index.js
+++ b/node_modules/circular-dependency-plugin/index.js
@@ -1,11 +1,10 @@
 let path = require('path')
-let extend = require('util')._extend
 let BASE_ERROR = 'Circular dependency detected:\r\n'
 let PluginTitle = 'CircularDependencyPlugin'
 
 class CircularDependencyPlugin {
   constructor(options) {
-    this.options = extend({
+    this.options = Object.assign({
       exclude: new RegExp('$^'),
       include: new RegExp('.*'),
       failOnError: false,
